bundles/org.openhab.binding.modbus.helioseasycontrols/README.md

```diff
@@ -0,0 +1,306 @@
+| infosMsg                         | general         | Infos as string                                                  | String                   |      |                                                                                                      |
+| statusFlags                      | general         | Status flags                                                     | String                   |      |                                                                                                      |
+| sensorConfigKwlFtf1              | general         | Sensor configuration (installed or not) KWL-FTF 1                | Switch                   |      | OFF = no sensor, ON = sensor installed                                                               |
+| sensorConfigKwlFtf2              | general         | Sensor configuration (installed or not) KWL-FTF 2                | Switch                   |      | OFF = no sensor, ON = sensor installed                                                               |
+| sensorConfigKwlFtf3              | general         | Sensor configuration (installed or not) KWL-FTF 3                | Switch                   |      | OFF = no sensor, ON = sensor installed                                                               |
+| sensorConfigKwlFtf4              | humidityControl | Sensor configuration (installed or not) KWL-FTF 4                | Switch                   |      | OFF = no sensor, ON = sensor installed                                                               |
+| sensorConfigKwlFtf5              | humidityControl | Sensor configuration (installed or not) KWL-FTF 5                | Switch                   |      | OFF = no sensor, ON = sensor installed                                                               |
+| sensorConfigKwlFtf6              | humidityControl | Sensor configuration (installed or not) KWL-FTF 6                | Switch                   |      | OFF = no sensor, ON = sensor installed                                                               |
+| sensorConfigKwlFtf7              | humidityControl | Sensor configuration (installed or not) KWL-FTF 7                | Switch                   |      | OFF = no sensor, ON = sensor installed                                                               |
+| sensorConfigKwlFtf8              | humidityControl | Sensor configuration (installed or not) KWL-FTF 8                | Switch                   |      | OFF = no sensor, ON = sensor installed                                                               |
+| globalManualWebUpdate            | humidityControl | Manual update of data (profile 8LXGP, XWP, firmware, SD files)   | Switch                   |      | ON = start update                                                                                    |

```

Comment:
This sounds like an [Action](https://www.openhab.org/docs/developer/bindings/#actions-bound-to-a-thing), rather a Channel.

Reply:
To be perfectly honest, I'm not really sure what this does exactly. The documentation doesn't really provide any details on this and the devices web interface (which is also based on the Modbus implementation) doesn't seem to support this as far as I could see.
You're probably right that this would also make sense as an Action.